mir-ci/mir_ci/test_display_configuration.py

```diff
@@ -0,0 +1,46 @@
+import asyncio
+import importlib
+import os
+from unittest.mock import ANY, Mock
+
+import pytest
+from mir_ci import SLOWDOWN, apps
+from mir_ci.display_server import DisplayServer
+from mir_ci.output_watcher import OutputWatcher
+
+short_wait_time = 1 * SLOWDOWN
+
+
+class TestDisplayConfiguration:
+    @pytest.mark.parametrize(
+        "local_server",
+        [
+            apps.mir_demo_server(),
+        ],
+    )
+    @pytest.mark.deps(pip_pkgs=("pyyaml",))
+    async def test_can_update_scale(self, local_server) -> None:
+        CONFIG_FILE = "/tmp/mir_ci_display_config.yaml"
+        try:
+            os.remove(CONFIG_FILE)
+        except OSError:
+            pass
```

Comment:
Best use a [tempfile.NamedTemporaryFile](https://docs.python.org/3/library/tempfile.html#tempfile.NamedTemporaryFile) as a context manager (you will need to "enter" it before it gets a name).